compiler/backend/src/org/jetbrains/kotlin/codegen/ClosureGenerationStrategy.kt

```diff
@@ -27,7 +27,8 @@ class ClosureGenerationStrategy(
 ) : FunctionGenerationStrategy.FunctionDefault(state, declaration) {
 
     override fun doGenerateBody(codegen: ExpressionCodegen, signature: JvmMethodSignature) {
-        initializeVariablesForDestructuredLambdaParameters(codegen, codegen.context.functionDescriptor.valueParameters)
+        initializeVariablesForDestructuredLambdaParameters(
```

Comment:
Minor. Can you, please, remove default argument from `initializeVariablesForDestructuredLambdaParameters`, since this was the only place, which used it?

Reply:
Yes! Done. :)